test/test_playbooks/lifecycle_environment.yml

```diff
@@ -1,78 +1,106 @@
 ---
-- hosts: localhost
+- hosts: fixtures
+  gather_facts: false
   tasks:
-  - name: Load server config
+  - name: 'Load server config'
     include_vars:
       file: server_vars.yml
-  - name: "Add a dev lifecycle environment"
-    katello_lifecycle_environment:
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+
+- hosts: tests
+  vars:
+    auth_params: &auth_params
       username: "{{ foreman_username }}"
       password: "{{ foreman_password }}"
       server_url: "{{ foreman_server_url }}"
       verify_ssl: "{{ foreman_verify_ssl }}"
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - name: "Add a dev lifecycle environment"
+    katello_lifecycle_environment:
```

Comment:
How about a `tasks/lifecycle_environment.yml`?

Reply:
Not sure about the benefit, but for the sake of consistency. ;) Will do